src/main/java/fish/payara/sample/debugging/model/ServiceModel.java

```diff
@@ -21,19 +24,26 @@ public ServiceModel() {
         this.libraries = new LinkedHashSet<>();
     }
 
+    /**
+     * @return all books across all libraries.
+     */
     public Collection<Book> getBooks() {
         return unmodifiableCollection(books);
     }
 
+    /**
+     * @return all libraries.
+     */
     public Collection<Library> getLibraries() {
         return libraries;
     }
 
-    public Book addBook(Library library, Book book) {
-        library.addBook(book);
-        return addBook(book);
-    }
-
+    /**
+     * Adds a book to the store.
```

Comment:
'Store' might be confusing as all the code refers to a 'library'.
```suggestion
     * Adds a book to the library.
```